rosdep/base.yaml

```diff
@@ -1350,6 +1355,11 @@ libexpat1-dev:
     portage:
       packages: [dev-libs/expat]
   ubuntu: [libexpat1-dev]
+libfcl-dev:
+  arch: [fcl]
+  debian: [libfcl-dev]
+  fedora: [fcl]
```

Comment:
It's better to leave it blank. That means that it's undefined and it will error. An empty list implies that nothing needs to be installed to resolve it and it will report successfully installed.